index.html

```diff
@@ -90,9 +90,19 @@
         </p>
         <pre class="idl">
           partial interface Navigator {
-            [SecureContext] Promise&lt;void&gt; share(ShareData data);
+            [SecureContext] Promise&lt;void&gt; share(optional ShareData data);
           };
         </pre>
+        <div class="note">
+          The <var>data</var> argument is marked as "optional", but really it
```

Comment:
Shane suggested I write a comment explaining why this is here, since it's quite unintuitive to have an optional argument that is mandatory.

In particular, I think while developers may not read the whole spec, the one thing that is fairly understandable to a non-web-spec person is IDL, so we may have confused developers reading the IDL and wondering why this is "optional". That's why I'm so strongly opposed to adding a misleading "optional" here, but if we are going to have it, I'd rather explain it.

If you'd like I can reduce the amount of text here and perhaps convert it into a `//` comment in the IDL itself rather than a note.